kitchen.yml

```diff
@@ -17,12 +17,23 @@ platforms:
   - name: debian-9
   - name: fedora-latest
   - name: freebsd-11
-  - name: freebsd-12
+  - name: freebsd-12.2
```

Comment:
Please make a comment here noting why you cannot use `freebsd-12` so we can fix it later.

Reply:
Ah I’d forgotten about this, the FreeBSD 12 bento box wants rebuilding and it looks like the ports version is out of sync with the kernel/base. It fails when running `pkg update` with an out of sync error. Not sure who deals with hole but it we can get that sorted we can revert this.